language/move-model/src/model.rs

```diff
@@ -2689,7 +2689,7 @@ pub struct FunctionData {
     type_arg_names: Vec<Symbol>,
 
     /// Specification associated with this function.
-    spec: Spec,
+    pub spec: Spec,
```

Comment:
Are you sure this is needed? We are using this information already outside of this module. so there must be a way to access it already?

Reply:
Good question. The reason for the `pub` here is not to read the `spec`, but to override the `spec`.

So, say when we drop an `aborts_if` in `new_spec` and want to override the original `spec` with the `new_spec`, we need to have a way to mutate `struct FunctionData`.

Reply:
Hmm... in order to update this, the entire global env needs to be mutable, which it currently is not (and currently you don't actually use the rewritten spec). Perhaps remove this for now until it is needed and we know how to deal with this.

Reply:
Discussed offline with @wrwg on where the `spec` is overriden (in `workflow.rs`) and why the `env` can be mutated (because we build a fresh `GlobalEnv` and re-run the bytecode pipeline whenever we try to remove one `aborts_if`).

The current design is highly inefficient and can benefit from a mutable (or partially mutable) `GlobalEnv`. But this is a much larger topic and is out of scope for this experimental tool.